app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ApplicationsController < ApplicationController
+  def new
+  end
+
+  def create
+    application = Applicant.create(application_params)
```

Comment:
I would use Applicant.new(application_params) here because it explicitly splits up the (new) and the (save) functionality. there is some confusing stuff that goes on under the hood of (create) and it can cause problems even though I think it looks like it works here.

Reply:
This is more of personal preference.